Examples/DicomImagePrintTags/Documentation.rst

```diff
@@ -1,14 +1,18 @@
 .. _lbl_print_image_meta_data_dictionary:
 
-Print Image Meta-Data Dictionary
-================================
+Read Image Meta-Data Dictionary and Print
+=========================================
 
 Overview
 --------
 
-This example illustrates how to read an image and access its meta-data dictionary. Most image types do not have a meta-data dictionary. The most common case for images with a dictionary is DICOM. For efficiency, the default reader settings will only load public DICOM tags (even group numbers). In the example we explicitly set the reader to load private tags (odd group numbers).
+This example illustrates how to read only an image's information and meta-data dictionary then access the meta-data dictionary via the `ImageFileReader <https://itk.org/SimpleITKDoxygen/html/classitk_1_1simple_1_1ImageFileReader.html>`_.
```

Comment:
Possibly change to: 
This example illustrates how to read the image's information and meta-data dictionary without loading the pixel content via the...